### ISSUE TITLE
Folder structuring

### ISSUE THREAD

**Issue by LukeSamkharadze** [tancredi/fantasticon#177]
I want to restructure my folder

**From:**
![image](https://user-images.githubusercontent.com/31849767/116248923-57293100-a77d-11eb-9153-0966850c2cb4.png)
**To:**
![image](https://user-images.githubusercontent.com/31849767/116248748-2a751980-a77d-11eb-9b3b-5f49ff45f545.png)

And still get the same `CSS` generated, currently `.svg` is not generated as `.icons-check`)

``` CSS
.icons-check:before {
    content: "\f105";
}
.icons-check-circle:before {
    content: "\f106";
}
```

I think this feature would be neat (Instead of `.svg` we could have used `_.svg` or something)

**Comment by justlampin** [comment]
👋 hi @LukeSamkharadze ! thanks for the suggestion!

there's currently a pending PR for this project here: https://github.com/tancredi/fantasticon/pull/161 that would allow you to do this.

here's an example:

**From:**
**Files:**
```
src
├── check-circle.svg
└── check.svg
```

**HTML output:**
![Screen Shot 2021-04-27 at 6 09 11 PM](https://user-images.githubusercontent.com/81541018/116324255-6dd38600-a785-11eb-8ca5-809fce761b7c.png)

**Comment by tancredi** [comment]
Hi @LukeSamkharadze - the `getIconId` feature is now merged and available in `v1.2.0`, but with some changes - `getIconId` will be called with a single `options` argument containing the `relativeFilePath` and other handy properties rather than just positional arguments, so you'll have to iterate a little bit over @justlampin  's solution (though it should be a pretty simple change)

Were you to have any issue with sorting this out, feel free to re-open :)

**Comment by LukeSamkharadze** [comment]
Thanks, Will try **ASAP**
